CONTRIBUTING.md

```diff
@@ -1,115 +1,144 @@
-   ```bash
-   ROSDISTRO=humble docker/run.sh
-   ```
-   Supported distributions include `humble` and `rolling`.
+<!-- omit in toc -->
+#### Before Submitting a Bug Report
 
-1. **[Optional]** Install pre-commit hooks. _This will probably cause you to not be able to create commits from your host machine since the hooks have dependencies that only exist in the development container._
-   ```bash
-   cd /ws/src/beluga
-   pre-commit install
-   ```
-   Alternatively, you can run the hooks manually.
-   ```bash
-   cd /ws/src/beluga
-   pre-commit run --all-files
-   ```
+A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information, and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.
 
-1. Create a working branch and start with your changes. The suggested branch name convention is `<user_name>/<feature_name>`.
+- Make sure that you are using the latest version.
+- Determine if your bug is really a bug and not an error on your side e.g. using incompatible versions (Make sure that you have read the [documentation](https://github.com/Ekumen-OS/beluga). If you are looking for support, you might want to check [this section](#i-have-a-question)).
+- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/Ekumen-OS/beluga/issues?q=label%3Abug).
+- Collect information about the bug:
+  - Stack trace (Traceback)
+  - OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+  - Version of the interpreter, compiler, middleware, package manager, depending on what is relevant.
+  - Possibly your input and the output
+  - Can you reliably reproduce the issue? And can you also reproduce it with older versions?
 
-1. Build and test the project.
-   ```bash
-   cd /ws
-   colcon build --symlink-install
-   colcon test
-   ```
+<!-- omit in toc -->
+#### How Do I Submit a Good Bug Report?
 
-1. <a name="running_an_example"></a>Run an example application using a pre-recorded rosbag.
+We use GitHub issues to track bugs and errors. If you run into an issue with the project:
 
+- Open an [Issue](https://github.com/Ekumen-OS/beluga/issues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the *reproduction steps* that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it's filed:
```

Comment:
nit:
```suggestion
After the issue has been submitted:
```

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/186/commits/88b19ea3d2839666e182802f81c167cf12ef4efc.